Casks/i/iexplorer.rb

```diff
@@ -8,13 +8,13 @@
   homepage "https://macroplant.com/iexplorer"
 
   livecheck do
```

Comment:
```suggestion
  # The response format of this URL seems to randomly switch between a plain
  # text changelog and a Sparkle appcast XML file. This matches the version
  # from dmg URLs in the text, which should work in either format.
  livecheck do
```

We should leave an explanatory comment here, as someone may try to switch this back to the `Sparkle` strategy otherwise.